documentation/src/docs/deployment/run-local.md

```diff
@@ -69,23 +70,69 @@ Download the [docker-compose.yml](https://github.com/pravega/pravega/tree/master
 wget https://raw.githubusercontent.com/pravega/pravega/master/docker/compose/docker-compose.yml
 ```
 
-We need to set the IP address of our local machine as the value of HOST_IP in the following command.
+We need to set the IP address of our local machine as the value of `HOST_IP` in the following command.
 ```
 HOST_IP=1.2.3.4 docker-compose up
 ```
 Clients can then connect to the controller at `${HOST_IP}:9090`.
 
 ## Configuring standalone
 
-The configuration properties for standalone can be specified in `config/standalone-config.properties`.
+Configure standalone server to communicate using SSL/TLS. To do so, edit the TLS-related properties in `standalone-config.properties` as shown below:
 
-These properties include ports for zookeeper, segment store and controller. They also contain other configurations related to security.
+  ```java
+  singlenode.enableTls=true
+  singlenode.keyFile=../config/key.pem
+  singlenode.certFile=../config/cert.pem
+  singlenode.keyStoreJKS=../config/standalone.keystore.jks
+  singlenode.keyStoreJKSPasswordFile=../config/standalone.keystore.jks.passwd
+  singlenode.trustStoreJKS=../config/standalone.truststore.jks
+
+  ```
+These properties include ports for Zookeeper, Segment Store and Controller. They also contain other configurations related to security.
 
 ## Running standalone with encryption and authentication enabled
 
 The configurations, `singlenode.enableTls` and `singlenode.enableauth` can be used to enable encryption and authentication respectively.(By default both these settings are disabled).
 
 In case `enableTls` is set to true, the default certificates provided in the `conf` directory are used for setting up TLS. These can be overridden by specifying in the properties file.
 
+1. Ensure that the server's certificate is trusted. If you run `/gradlew startStandalone` without it, you'll encounter the following error:
```

Comment:
It's `./gradlew startStandalone` (you need the `.` at the beginning)

Reply:
@andreipaduroiu Accepted suggestion